w3-hw1-vy/cypress/fixtures/account.json

```diff
@@ -0,0 +1,117 @@
+{
+  "user": [
```

Comment:
I see that you store data for happy and unhappy cases in the same array, it's not wrong but it may cause confuse for others or even you when you maintain scripts in the future. One example:
```json
{
  "email": {
    "valid": ["abc@gmail.com", "a12@yahoo.com"],
    "invalid": ["@@@", "abc...."]
  }
}
```
You can know that your data is for happy or unhappy case right away